host/4/bullseye/amd64/base/sharedconfig/start.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+export DOTNET_USE_POLLING_FILE_WATCHER=true
```

Comment:
This script? Or the #!/usr/bin/env bash?  
Script starts host. 
The other thing tells linux what style of bash to expect in the script

Reply:
No this option `DOTNET_USE_POLLING_FILE_WATCHER` mean. :)

Reply:
if set dotnet filewatcher will actively poll for filesystem changes, if not set, it will rely on file change notifications (which doesnt always work in some disk setups)